### DISCUSSION TITLE
Can't get CMD-T on OSX to open new tab next to my current tab/child tab

### DISCUSSION THREAD

**Discussion by mfn** [piroor/treestyletab#3141] [Q&A] [answered]
Hi 👋🏼 ,

I've already searched and read through https://github.com/piroor/treestyletab/discussions/3076 and similar discussion but… nothing works 🤷🏼 

I'm on FF 101.0 (64-bit) on 10.15.7 using TST 3.8.24 and wanted to try changing the new tabs open next the my current tab (right now they always open at the end), which is VERY FAR down in my case, having 40+ tabs open.

I'm starting with this setting, but nothing changed?
![image](https://user-images.githubusercontent.com/87493/172811745-c65a6037-588d-4c94-8a19-3b2b2c3f1fac.png)

I trigger it via <kbd>cmd</kbd>-<kbd>t</kbd>.

My general settings are like this:
![image](https://user-images.githubusercontent.com/87493/172811907-3947938f-2bd1-4070-a4a6-729b6a83ae4a.png)

I also played around with these settings:
![image](https://user-images.githubusercontent.com/87493/172811969-72cf2864-cfe9-4bad-b67c-059be4c6d62c.png)

But no matter what I tried, it didn't seem to work…

My other extensions are:
- AWS Console Lanels
- Firefox Multi-Account Containers
- Gmail Show Time
- Reddit Enhancement Suite
- Snap Links
- Tree Style Tab 🎉 
- uBlock Origin

I also tried disabling all of them except TST, no change.

Any ideas? 🤔 Thanks 🙏🏼

**Comment by irvinm** [comment, marked as answer]
Have you tried setting:

![image](https://user-images.githubusercontent.com/979729/173161976-7d0223d0-b688-42cd-a543-478d3e73cbc1.png)

**Reply by mfn** [reply]
# 👀 

works…

For reference:
- `browser.tabs.insertAfterCurrent`
- https://www.reddit.com/r/firefox/comments/p2lzz1/why_is_browsertabsinsertaftercurrent_not_exposed/

I had no idea. And now this isn't even TST related.

Thank you 🙏🏼

**Reply by mfn** [reply]
I came back to _once more_ say thank you! Now having worked with this setting for a week, it's an absolute game changer for my tab hording habit and max navigation and finding tabs _so much easier_.
